.github/workflows/release.yml

```diff
@@ -98,15 +98,15 @@ jobs:
   deploy_stage:
     name: "Deploy: Stage"
     needs: [ build, version, deploy_sqa]
-    if: ${{ needs.version.outputs.should_release == 'true' && needs.version.outputs.prerelease == 'false'}}
+    if: ${{ needs.version.outputs.should_release }} == 'true' && ${{ needs.version.outputs.prerelease }} == 'false'
     uses: ./.github/workflows/_skaffold-deploy-k8s.yml
     secrets: inherit
     with:
       environment: "jax-cluster-prod-10--stage"
```

Comment:
Is the environment name correct? "jax-cluster-prod-10--stage". I see two '-'  before the word stage.

Reply:
Yes, they should all have the two `-` there.

The format is `$CLUSTER_NAME--$ENV_NAME` e.g.  `jax-cluster-prod-10--stage` and `jax-cluster-dev-10--sqa`